src/Standards/Squiz/Docs/WhiteSpace/MemberVarSpacingStandard.xml

```diff
@@ -0,0 +1,142 @@
+<documentation title="Member Var Spacing">
+    <standard>
+    <![CDATA[
+    There should be one blank line before the first member var.
```

Comment:
```suggestion
    There should be one blank line before the first member variable.
```

Here and elsewhere in the docs.

I suspect this sniff stems from the time when the `var` keyword was still used for properties and in those days, "member var" would be clear, but nowadays, it's not as obvious what that means.

I can even imagine using the term property throughout the docs (except for in the title) as that's the more common term nowadays.

Something like:
```suggestion
    There should be one blank line before the first property (member variable).
```

(with the bit between brackets only being used in the first standard block)

Reply:
I have to say it did feel strange not referring to them as properties! Thanks, I've addressed this in 34203fa67